### ISSUE TITLE
Cloud Anchor Ipad not supported

### ISSUE THREAD

**Issue by philmaas** [google-ar/arcore-unity-sdk#229]
I know from the docs Ipad is not yet supported. 
The cloud anchor sample compiles fine but can't host an anchor during runtime. 

Curious why exactly ipad is not supported and when we can expect support for all arkit devices.

**Comment by Jacquelinett** [comment]
It doesn't seem like it just an iPad issue. I tried running the sample ios kit on both the latest iPad pro and an iPhone 8 and they both compiled fine, but both get stuck at "Hosting..." or "Resolving..." depend on which you pick.

I made sure to follow the provided iOS guide step by step, and redo it 3 times already with no result. Checking the filebase's real-time database show that it did receive the room number + timestamp, but no information about the map of a location or anything.

Any help would be appreciated
